### PR TITLE
HSEARCH-2274 Making sure ES host can be specified using property "hib…

### PR DESCRIPTION
**Pull request by gunnarmorling**
…ernate.search.default.elasticsearch.host"

### CODE REVIEW

File: elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/client/impl/JestClient.java [hunk 1]
```diff
@@ -47,8 +47,18 @@
...
 
+	/**
+	 * HTTP response code for a request timed out.
+	 */
 	private static final int TIME_OUT = 408;
 
+	/**
+	 * Prefix for accessing the {@link ElasticsearchEnvironment#SERVER_URI} variable. That's currently needed as we
+	 * don't access this one in the context of a specific index manager. The prefix is used to have the property name
+	 * inline with the other index-related property names, it cannot be overridden per moment, though.
```
Comment (gsmet) [inline elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/client/impl/JestClient.java]: "inline with" -> in line with
"per moment" -> per index? at the moment?

File: elasticsearch/src/test/java/org/hibernate/search/backend/elasticsearch/test/HostCanBeConfiguredIT.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+import org.hibernate.boot.Metadata;
+import org.hibernate.boot.MetadataSources;
+import org.hibernate.boot.registry.StandardServiceRegistryBuilder;
+import org.hibernate.search.exception.SearchException;
+import org.hibernate.search.testsupport.TestForIssue;
+import org.junit.Test;
+
+/**
+ * Test to make sure that the ES host can be configured and not the default is applied unconditionally. To test it, the
```
Comment (gsmet) [inline elasticsearch/src/test/java/org/hibernate/search/backend/elasticsearch/test/HostCanBeConfiguredIT.java]: Looks like the "not" is misplaced.

Maybe "and the default is not applied unconditionally" would be better?

**Comment by gsmet** [comment]
@gunnarmorling looks good to me apart from the minor remarks on the comments. I let you fix them and merge? Thanks!

**Comment by Sanne** [comment]
merged with @gsmet 's suggestions

**Comment by gunnarmorling** [comment]
Thanks, everyone!

**Pull request closed without merging**